Index: jessie-stage2/rootfs/etc/ssh/sshd_config
===================================================================
--- jessie-stage2.orig/rootfs/etc/ssh/sshd_config
+++ jessie-stage2/rootfs/etc/ssh/sshd_config
@@ -108,7 +108,7 @@
 #Banner none

 # Allow client to pass locale environment variables
-AcceptEnv LANG LC_*
+#AcceptEnv LANG LC_*

 # override default of no subsystems
 Subsystem	sftp	/usr/lib/openssh/sftp-server
